src/app.js

```diff
@@ -316,10 +318,25 @@ class Messenger extends EventEmitter {
     // The 'payload' param is a developer-defined field which is set in a postback
     // button for Structured Messages.
     const payload = event.postback.payload;
-
     debug("onPostback for user:%d with payload '%s'", senderId, payload);
+    this.emit('postback', {event, senderId, session, payload});
+
+    if (this.options.emitGreetings && this.greetings.test(payload)) {
```

Comment:
I suppose this block (lines 324-336) could be moved to a standalone function since it is essentially a duplicate block of what is in `onMessage` (lines 258-270). I overlooked that when I submitted the PR.